app/src/main/java/net/schueller/peertube/fragment/VideoOptionsFragment.java

```diff
@@ -95,4 +97,15 @@ public View onCreateView(LayoutInflater inflater,
 
     }
 
+    private String getCurrentVideoQuality(ArrayList<File> files) {
+        SharedPreferences sharedPref = PreferenceManager.getDefaultSharedPreferences(getContext());
+        Integer videoQuality = sharedPref.getInt("pref_quality", 0);
+
+        for (File file : files) {
+            if (videoQuality.equals(file.getResolution().getId())) {
+                return file.getResolution().getLabel();
+            }
+        }
+        return "Auto";
```

Comment:
Should I return "Auto" here? Or something else?

Reply:
Although auto has not been implemented yet I think it makes sense to return that, but move the string into an xml so it can be translated.

Reply:
Done.